src/group_theory/perm/basic.lean

```diff
@@ -287,6 +287,18 @@ equiv.ext $ λ ⟨x, hx⟩, by { dsimp [subtype_perm, of_subtype],
 
 @[simp] lemma default_perm {n : Type*} : default (equiv.perm n) = 1 := rfl
 
+/-- Permutations on a subtype are equivalent to permutations on the original type that fix pointwise
+the rest. -/
+protected def subtype_equiv (p : α → Prop) [decidable_pred p] :
```

Comment:
```suggestion
@[simps]
protected def subtype_equiv (p : α → Prop) [decidable_pred p] :
```
although you should check what that generates with `simps?`

Reply:
It gives
```
[simps] > adding projection equiv.perm.subtype_equiv_apply_coe:
        > ∀ {α : Type u} (p : α → Prop) [_inst_1 : decidable_pred p] (f : perm (subtype p)),
  ↑(⇑(perm.subtype_equiv p) f) = ⇑of_subtype f
[simps] > adding projection equiv.perm.subtype_equiv_symm_apply:
        > ∀ {α : Type u} (p : α → Prop) [_inst_1 : decidable_pred p]
(f : {f // ∀ (a : α), ¬p a → ⇑f a = a}), ⇑((perm.subtype_equiv p).symm) f = ↑f.subtype_perm _
```
which I assume to be fine, although I'm not so knowledgeable about the naming convention here.

Reply:
Those look good to me.